plugins/woocommerce-admin/client/marketing/hooks/useInstalledPlugins.ts

```diff
@@ -18,22 +21,36 @@ export type UseInstalledPlugins = {
 
 /**
  * Hook to return plugins and methods for "Installed extensions" card.
+ *
+ * The list of installed plugins will not include registered and recommended marketing channels.
```

Comment:
> From the perspective of this hook, it's named `useInstalledPlugins` but filters out some plugins internally, which might surprise the hook users.
> 
> While this can be clarified by this code comment, it might be possible to rename this hook, or to add hook parameters to indicate whether filtering out the registered and/or recommended ones.

You got a point. I called it `useInstalledPlugins` because we uses the classic marketing wp.data store "wc/marketing" and it uses the term "plugin". I'm thinking I can rename them to `useInstalledExtensions` instead, because it is meant to be used with the "Installed extensions" card. Similarly, I'm thinking to rename `useRecommendedPlugins` to `useDiscoverTools`.

> Another idea is to keep exposing the original **`installedPlugins`** and also exposing `installedPluginsWithoutChannels` separately.

I less prefer this idea because we won't be using the `installedPlugins`.

Reply:
> I'm thinking I can rename them to `useInstalledExtensions` instead, because it is meant to be used with the "Installed extensions" card.

Renaming to `useInstalledExtensions` perhaps makes it more concrete in association with the "Installed extensions" card but it seems to still have the same confusion in the `useInstalledPlugins` naming: The `useInstalledExtensions()` statement still doesn't look like it will filter out some plugins.